lib/client.js

```diff
@@ -1001,25 +1001,24 @@ client.prototype.handleMessage = function handleMessage(message) {
 				else {
 					// Message is an action (/me <message>)..
 					var actionMessage = _.actionMessage(msg);
-					if(actionMessage) {
-						message.tags["message-type"] = "action";
-						this.log.info(`[${channel}] *<${message.tags.username}>: ${actionMessage[1]}`);
-						this.emits(["action", "message"], [
-							[channel, message.tags, actionMessage[1], false]
-						]);
+					message.tags["message-type"] = actionMessage ? "action" : "chat";
+					messageContent = actionMessage ? actionMessage[1] : msg;
```

Comment:
Changes done. Sorry for that ;-)

Reply:
I mean reassign `msg` to `actionMessage[1]` instead of creating `messageContent` for this.
```js
if(actionMessage) {
    msg = actionMessage[1];
}
```
or
```js
msg = actionMessage ? actionMessage[1] : msg;
```